### PR TITLE
feat: control displayed sponsors

### PR DESCRIPTION
**Pull request by Ritabear**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [X] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.

- add a new variable is_show to manage sponsor data retrieval via the sponsor API.
- testing for a new feature implementation.

issue related: [#508](https://github.com/pycontw/pycontw-frontend/issues/508)

### CODE REVIEW

File: src/sponsors/models.py [hunk 1]
```diff
@@ -23,6 +23,12 @@ class Sponsor(ConferenceRelated):
         verbose_name=_('name'),
         max_length=100,
     )
+
+    is_shown = models.BooleanField(
+        verbose_name=_('is shown'),
+        default=False,
```
Comment (mattwang44) [inline src/sponsors/models.py:29]: Q: default 為 true or false 哪一個對贊助組的操作會比較方便？

File: src/sponsors/tests/test_api.py [hunk 2]
```diff
@@ -43,6 +44,7 @@ def test_should_get_sponsor_data(self, api_client):
         # small-order > large-order > null-order
         gold_sponsor_names = [s['name_en_us'] for s in gold_sponsors]
         assert gold_sponsor_names == ['4', '2', '3']
+        assert '5' not in gold_sponsor_names
```
Comment (mattwang44) [inline src/sponsors/tests/test_api.py:47]: 這兩行意義有點重疊，感覺可以不用加？

File: src/sponsors/models.py [hunk 3]
```diff
@@ -23,6 +23,12 @@ class Sponsor(ConferenceRelated):
         verbose_name=_('name'),
         max_length=100,
     )
+
+    is_shown = models.BooleanField(
+        verbose_name=_('is shown'),
```
Comment (mattwang44) [inline src/sponsors/models.py:28]: 該詞在 Django admin 上要顯示的翻譯呢？

hint: [makemessages command](https://github.com/pycontw/pycon.tw/blob/03d3e8af584d4b3d3bcbdd23fcf05d6da70f93de/document/deploy_docker_dev.md#L31-L32)

File: src/sponsors/models.py (hunk 1 above)
Comment (Ritabear) [reply src/sponsors/models.py:29]: 我覺得是false耶～
因為不想所有東西都出去，出去前通常會在檢查一次＠＠

Comment (mattwang44) [reply src/sponsors/models.py:29]: 沒把握的話也可以問問贊助組看看~

**Review by mattwang44 — APPROVED** [review]
LGTM

**Comment by SivanYeh** [comment]
I tested it on staging and found previous logos would disappear. 
Pasted Graphic 1

Is it normal? Should we inform sponsor team to recreate logos for new control?

**Pull request merged**